compose.yml

```diff
@@ -22,7 +22,7 @@ services:
     image: otel/opentelemetry-collector
     command: [ "--config=/etc/otelcol-config.yml" ]
     volumes:
-      - ./src/otelcollector/otelcol-config.yml:/etc/otelcol-config.yml
+      - ./otelcol-config.yml:/etc/otelcol-config.yml
```

Comment:
i think you can keep ./src part.

Reply:
to avoid too many files in the root

Reply:
I'm with @cijothomas in this one.
I'd rather keep the otelcol-config within the otelcol folder.

Any reason you would like to move to the root @mic-max ?

Reply:
It just felt like a more top level config similar to the docker compose file. I'm fine to close this PR though and leave as is too - just wanted to spur a little discussion :)